libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/speedlimit/SpeedInfoBehavior.kt

```diff
@@ -0,0 +1,19 @@
+package com.mapbox.navigation.dropin.speedlimit
+
+import com.mapbox.navigation.ui.speedlimit.model.SpeedInfoValue
+import kotlinx.coroutines.channels.BufferOverflow
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.asSharedFlow
+
+internal class SpeedInfoBehavior {
```

Comment:
This class is almost the same as `MapClickBehavior`. All other click behaviors are likely going to be implemented the same way. Which means we can have a single generic class:
```kotlin
internal class ClickBehavior<T> {

    private val _clickBehavior = MutableSharedFlow<T>(
        extraBufferCapacity = 1,
        onBufferOverflow = BufferOverflow.DROP_OLDEST,
    )
    val clickBehavior = _clickBehavior.asSharedFlow()

    fun onClicked(value: T) {
        _clickBehavior.tryEmit(value)
    }
}
```